src/service-manager.js

```diff
@@ -221,10 +248,30 @@ class ServiceManager {
     )
   }
 
+  /**
+   * Updates an account specified by parameter name on a ledger specified by parameter ledger.
+   * @param  {String} ledger  Ledger prefix identifying the ledger that holds the updated account.
+   * @param  {String} name    Name of the updated account.
+   * @param  {JSON}   options Used to specify the balance to which the account is updated. Use options.balance.
+   * @return {[Promise]}
+   */
   updateAccount (ledger, name, options) {
     return co.wrap(this._updateAccount).call(this, ledger, name, options || {})
   }
 
+  * _updateKitAccount (ledgerPrefix, username) {
+    const db = yield this._getLedgerDb(ledgerPrefix)
+    // const now = Math.round(new Date().getTime() / 1000)
+    yield db.run(
+      'INSERT OR REPLACE INTO Users (username, created_at, updated_at) VALUES (?, ?, ?)',
+      [ username, '2017-01-10 16:12:17.039 +00:00', '2017-01-10 16:12:17.039 +00:00' ]
```

Comment:
The database column is defined to be `NOT NULL`. On the other hand, I don't need the timestamps in the test logic, so any timestamp is fine.